src/ServiceControl.AcceptanceTests/MessageFailures/When_a_pending_retry_is_retried_by_queue_and_timeframe.cs

```diff
@@ -43,7 +44,11 @@ public void Should_succeed()
 
                     if (failedMessage.Status == FailedMessageStatus.RetryIssued)
                     {
-                        return true;
+                        FailedMessage[] failedMessages;
+                        if (TryGet("/api/errors", out failedMessages))
```

Comment:
We need this to make sure that `FailedMessageIndex` is up to date. Otherwise it's possible that the `FailedMessage` document is returned via `/api/errors/{msgid}` and retried via. `/api/errors/{msgid}/retry`  but when we call retry from `/api/pendingretries/queues/retry` the `FailedMessageIndex` did not catch-up. No `FailedMessageRetry` is deleted and who retry becomes no-op.